perfscale_regression_ci/scripts/scalability/replace_nodes/clouds/aws.sh

```diff
@@ -0,0 +1,11 @@
+source ./common.sh
+
+export OPENSHIFT_INFRA_NODE_VOLUME_SIZE=128
```

Comment:
I didn't see INFRA related ENV vars used anywhere, but it's fine if you think we will use it in the future.